docs/chain/collator.md

```diff
@@ -405,7 +405,40 @@ The corresponding methods can be found as an extrinisc under `parachainStaking >
 
 ## Monitoring
 
-TODO
+It would be ideal if monitoring is done from a different host however in cases of resource constraints it would be better to run these monitoring tools on the same machine where your collator is running than not having.The following Monitoring stack is recommended.
+You can either run grafana by your own or subscribe free cloud option here https://grafana.com/products/cloud/. 
+NB: if you want to use cloud option you need to expose prometheus to outside access by chaning the ports section to ```  - "9090:9090"``` and reverse proxying with webserver is recommended.
+
+- Prometheus
+- Grafana
+- Node exporter
+- Slack
+
+There are two types of metrics we are going to collect. **Node Metrics** and **Blockchain Metrics.**
+
+### Prerequisite
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo chmod +x /usr/local/bin/docker-compose
+sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+```
```

Comment:
Might be best to link to the official docker compose guide? We would need to update if there is a new version or anything.